compiler/llvm/llvmDebug.cpp

```diff
@@ -567,6 +568,50 @@ llvm::DIType* DebugData::constructTypeFromChplType(llvm::Type* ty, Type* type) {
     type->symbol->llvmDIType = N;
     return N;
 
+  } else if (type == dtLocaleID) {
+    // handle locale types
+    llvm::SmallVector<llvm::Metadata *, 1> EltTys;
+    llvm::Type* nodeTy = dtInt[INT_SIZE_32]->getLLVMType();
+    EltTys.push_back(dibuilder->createMemberType(
+      defInfo.maybeScope(),
+      "node",
+      defInfo.file(),
+      defInfo.line(),
+      layout.getTypeSizeInBits(nodeTy),
+      8*layout.getABITypeAlign(nodeTy).value(),
+      0, /* offset, assume its zero */
+      llvm::DINode::FlagZero,
+      getType(dtInt[INT_SIZE_32])
+    ));
+    // TODO: we should more directly check for sublocales, not just use
+    // the gpu locale model as a proxy for that
+    if (usingGpuLocaleModel()) {
+      EltTys.push_back(dibuilder->createMemberType(
+        defInfo.maybeScope(),
+        "subloc",
+        defInfo.file(),
+        defInfo.line(),
+        layout.getTypeSizeInBits(nodeTy),
+        8*layout.getABITypeAlign(nodeTy).value(),
+        layout.getTypeSizeInBits(nodeTy), /* offset, assume after node */
+        llvm::DINode::FlagZero,
```

Comment:
I see we do similar things below for other types, but how do we know this is the alignment of the locale type? Can we be sure?